src/components/TransactionsTabContent/TransactionsTabContent.tsx

```diff
@@ -20,6 +20,8 @@ export interface TransactionsTabContentProps {
 }
 
 export const TransactionsTabContent = ({ transactions, TxDialog, EditDialog }: TransactionsTabContentProps) => {
+  const showRecordTransaction = false;
```

Comment:
Please add jsdoc, that we're hiding record transaction button and dialog, until spv-wallet functionality for recording transactions would fulfil users needs and expectations.

Reply:
Sure.